### ISSUE TITLE
zstd compression support

### ISSUE THREAD

**Issue by ernsteiswuerfel** [vaeth/zram-init#10]
Would really be nice to have zstd compression support, since zstd is now included in >= kernel 4.14.x!

**Comment by vaeth** [comment]
Some implementation of zstd is available in the kernel, but to my knowledge this is not used by zram (yet).

**Comment by ernsteiswuerfel** [comment]
You are right. Forgot for a moment that this is a prerequisite for zram-init being able to use it... Sorry!

**Comment by olifre** [comment]
Should be usable by zram starting with the next kernel release:
https://github.com/torvalds/linux/commit/5ef3a8b12556d7fcba81edc74e9d85b029615ae0#diff-4caba8d67df6c1280c662f0552c6021e

**Comment by vaeth** [comment]
Thanks for the information.

From zram-init side there is not much to do: The new algorithm can just be specified.
So presumably only the information/example configs will be updated when the new kernel is officially released (I suppose in January or so.)

I also thought about using zstd by default, but maybe it is better to not touch the kernel's algorithm choice (which presumably depends on .config) unless the user explicitly requires it.

**Comment by ernsteiswuerfel** [comment]
Thanks!
